gensim/sklearn_integration/sklearn_wrapper_gensim_lsimodel.py

```diff
@@ -58,7 +58,7 @@ def set_params(self, **parameters):
     def fit(self, X, y=None):
         """
         For fitting corpus into the class object.
-        Calls gensim.model.LsiModel:
+        Calls gensim.models.LsiModel:
         >>>gensim.models.LsiModel(corpus=corpus, num_topics=num_topics, id2word=id2word, chunksize=chunksize, decay=decay, onepass=onepass, power_iters=power_iters, extra_samples=extra_samples)
```

Comment:
This documentation line doesn't seem to help -- what are these undefined variables like `id2word`, `chunksize` etc?